rfc6962-bis.xml

```diff
@@ -1231,18 +1293,22 @@ but it is expected there will be a variety.
+        </t>
+        <t>
+          TODO: What should the TLS client communicate in the extension_data? Version(s) of CT that it supports? Certain types of TransItem that it can handle? Whether or not it wants to gossip?
         </t>
         <t>
-          In addition to normal validation of the certificate and its chain, TLS clients SHOULD validate each SCT by computing the signature input from the SCT data as well as the certificate and verifying the signature, using the corresponding log's public key. TLS clients MUST reject SCTs whose timestamp is in the future.
+          In addition to normal validation of the certificate and its chain, TLS clients SHOULD validate each supplied SCT by computing the signature input from the SCT data as well as the certificate and verifying the signature, using the corresponding log's public key. TLS clients MUST reject SCTs whose timestamp is in the future.
         </t>
         <t>
-          By validating SCTs, TLS clients can thus determine whether certificates are compliant. A certificate not accompanied by a valid SCT MUST NOT be considered compliant by TLS clients. However, specifying the TLS clients' behavior once compliance or non-compliance has been determined (for example, whether a certificate should be rejected due to the lack of valid SCTs) is outside the scope of this document.
+          TLS clients SHOULD also validate each supplied inclusion proof (see <xref target="verify_inclusion"/>), in order to audit the log. If no inclusion proof was supplied by the TLS server, the TLS client MAY request one directly from the corresponding log using <spanx style="verb">get-proof-by-hash</spanx> (<xref target="get-proof-by-hash"/>) or <spanx style="verb">get-all-by-hash</spanx> (<xref target="get-all-by-hash"/>), and then validate it.
         </t>
         <t>
-          A TLS client MAY audit the corresponding log by requesting, and
-          verifying, a Merkle audit proof for said certificate.
-          If the TLS client holds an STH that predates the SCT, it MAY, in the process of auditing, request a new STH from the log (<xref target="fetch_sth"/>), then verify it by requesting a consistency proof (<xref target="fetch_consistency"/>).
+          By validating SCTs and inclusion proofs, TLS clients can thus determine whether certificates are compliant. A certificate not accompanied by a valid SCT or inclusion proof MUST NOT be considered compliant by TLS clients. However, specifying the TLS clients' behavior once compliance or non-compliance has been determined (for example, whether a certificate should be rejected due to non-compliance) is outside the scope of this document.
```

Comment:
Addressed in https://github.com/robstradling/certificate-transparency-rfcs/commit/d6027da0a918e86ae20860b0aa54b38b8d3a3b7f